src/transaction/input/multisig.js

```diff
@@ -142,30 +145,31 @@ MultiSigInput.prototype.isValidSignature = function(transaction, signature) {
  * @param {Input} input
  * @returns {TransactionSignature[]}
  */
-MultiSigInput.normalizeSignatures = function(transaction, input, inputIndex, signatures, publicKeys) {
-  return publicKeys.map(function (pubKey) {
-    var signatureMatch = null;
-    signatures = signatures.filter(function (signatureBuffer) {
+// eslint-disable-next-line max-len
+MultiSigInput.normalizeSignatures = function (transaction, input, inputIndex, signatures, publicKeys) {
```

Comment:
I really dislike function declarations that are broken into multiple lines and I decided to disable this error. @ClemensLey thoughts on this?

There are a few cases like this in this pull request.

Reply:
I think that when we add flow we should break long function declarations into multiple lines (they become long quite frequently when you have type declarations in there). Until then we can just disable the rule for the affected functions like you do here.